console/backend/src/main/java/org/frankframework/management/web/ConfigurationsEndpoint.java

```diff
@@ -141,15 +153,15 @@ public ResponseEntity<?> getConfigurationDetailsByName(@PathVariable("configurat
 	@Relation("configuration")
 	@Description("change the active configuration version, and optionally schedule or load it directly")
 	@PutMapping(value = "/configurations/{configuration}/versions/{version}", produces = MediaType.APPLICATION_JSON_VALUE, consumes = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<?> manageConfiguration(
-			@PathVariable("configuration") String configurationName,
-			@PathVariable("version") String encodedVersion,
-			@RequestParam(value = "datasourceName", required = false) String datasourceName,
-			@RequestBody Map<String, Object> json) throws ApiException {
+	public ResponseEntity<?> manageConfiguration(@PathVariable("configuration") String configurationName,
+												 @PathVariable(HEADER_VERSION) String encodedVersion,
+												 @RequestParam(value = "datasourceName", required = false) String datasourceName,
+												 @RequestBody Map<String, Object> json) throws ApiException {
 
 		RequestMessageBuilder builder = RequestMessageBuilder.create(BusTopic.CONFIGURATION, BusAction.MANAGE);
 		builder.addHeader(BusMessageUtils.HEADER_CONFIGURATION_NAME_KEY, configurationName);
-		builder.addHeader("version", HttpUtils.urlDecode(encodedVersion));
+		builder.addHeader(HEADER_VERSION, HttpUtils.urlDecode(encodedVersion));
```

Comment:
Tja, ik snap dat je IDE zegt dat je dezelfde waarde meer dan 3 keer gebruikt, maar misschien is het wel goed onderscheid te maken tussen de REST path variable en BUS header naam.